File: Bag.m
//
//  Bag.m
//  MacPuf
//
//  Created by J. Michael Dean on Wed Mar 06 2002.
//  More massive coding changes between December and February, 2003
//  Copyright (c) 2002 - 2003 J.Michael Dean, M.D., M.B.A. All rights reserved.file:///Library/Developer/Documentation/DocSets/com.apple.ADC_Reference_Library.DeveloperTools.docset/Contents/Resources/Documents/documentation/DeveloperTools/Conceptual/XcodeCoreDataTools/Art/multiItemappearance.jpg
//

#import "Bag.h"


@implementation Bag

-(id) init
{
    NSLog(@"Creating %@", self);    
    if (self = [super init]) {
        volume = 0.000;
        carbonDioxide = 0.000;
        oxygen = 0.000;
        NSLog(@"Object initialized: %@", self);
        }
    return self;
}

-(NSString *) description
{
    return [NSString stringWithFormat:
        @"\nThere is no Douglas bag attached at present."];
}

@end

File: Bag.swift
//
//  Bag.swift
//  MacPuf_IPad
//
//  Created by J Michael Dean on 4/11/15.
//  Copyright (c) 2015 J Michael Dean. All rights reserved.
//
//	Models a Doublas bag for rebreathing experiments.  I did not implement
//	this at all in the previous Objective C attempt, but this is worth doing.

import Foundation

class Bag {
	
	var volume:Double = 0.0						// MacPuf variable BAG   Factor 116
	var carbonDioxide:Double = 0.0		// MacPuf variable BAGC  Factor 38
	var oxygen:Double = 0.0						// MacPuf variable BAGO  Factor 37
	
	init(volume:Double, carbonDioxide:Double, oxygen:Double){
		self.volume = volume
		self.carbonDioxide = carbonDioxide
		self.oxygen = oxygen
	}
}